pages/[walletAddress]/fund-wallet.tsx

```diff
@@ -38,15 +94,38 @@ const TopUpPage = () => {
       </div>
       <main className="min-h-fit grow">
         <div className="mx-auto min-h-full max-w-7xl px-4 py-6 sm:px-6 lg:px-8">
+          {PREFERED_NETWORK !== NetworkType.MAINNET && (
+            <div>
+              <h2 className="text-xl text-white">Buy crypto</h2>
+              <p className="mt-2 text-zinc-200">
+                Our provider {"doesn't"} support transferring to Tezos contract
+                yet. So after the transaction succeed, we will automatically
+                create a transaction from your wallet to your TzSafe wallet
+              </p>
+              <button
+                className="mt-4 rounded bg-primary px-4 py-2 font-medium text-white hover:bg-red-500 hover:outline-none focus:bg-red-500"
+                onClick={() => {
+                  wertWidgetRef.current.mount();
+                  setError(undefined);
+                }}
+              >
+                Open the on-ramp service
+              </button>
+              <p className="mt-2">{!!error && renderError(error, true)}</p>
+            </div>
+          )}
           {!state.currentContract ? (
             <h2 className="text-center text-xl text-zinc-600">
               Please select a wallet in the sidebar
             </h2>
           ) : (
-            <TopUp
-              address={state.currentContract ?? ""}
-              closeModal={() => {}}
-            />
+            <>
+              <h2 className="mt-12 text-xl text-white">Send to the contract</h2>
```

Comment:
```suggestion
              <h2 className="mt-12 text-xl text-white">Fund from {alias(logged_in_account)}</h2>
```
something like this

Reply:
I'm not sure to understand, do you mean showing the TzSafe alias?